lib/rules/headers/copyright.js

```diff
@@ -9,15 +9,23 @@ exports.check = function (sr, done) {
     if ($c.length) {
         var year = (sr.getDocumentDate() || new Date).getFullYear()
         ,   text = sr.norm($c.text())
-        ,   startRex = "^Copyright © (?:(?:199\\d|20[01]\\d)-)?" + year + " .*?W3C® \\(MIT, ERCIM, Keio, Beihang\\), "
-        ,   hatesFreedom = /All Rights Reserved/.test(text)
-        ,   evil = "All Rights Reserved\\."
-        ,   good = "Some Rights Reserved: this document is dual-licensed, CC-BY and W3C Document License\\."
-        ,   endRex = " W3C liability, trademark,? and document use rules apply\\.$"
-        ,   rex = new RegExp(startRex + (hatesFreedom ? evil : good) + endRex)
+        ,   startRex = "^Copyright © (?:(?:199\\d|20[01]\\d)-)?" + year + " .*?W3C® \\(MIT, ERCIM, Keio, Beihang\\)"
+        ,   dualLicensed = /CC-BY/.test(text);
+
+        // See https://lists.w3.org/Archives/Public/spec-prod/2015JanMar/0001.html
+        if (sr.getDocumentDate() < new Date(2015, 01, 07)) // Before 06 Feb 2015
+            var w3cOnlyStatement = ", All Rights Reserved";
```

Comment:
You're declaring this variable three times (and the three declarations get hoisted to the function's top). Just declare it as part of the top block and then simply assign to it in the conditional.